backend/fms_core/viewsets/process_measurement.py

```diff
@@ -42,6 +42,12 @@ class ProcessMeasurementViewSet(viewsets.ModelViewSet, TemplateActionsMixin):
             "template": SAMPLE_TRANSFER_TEMPLATE,
             "importer": TransferImporter,
         },
+        {
+            "name": "Sample Quality Control",
+            "description": "Upload the provided template with samples that underwent a quality control.",
+            "template": SAMPLE_QC_TEMPLATE,
+            "importer": SampleQCImporter,
```

Comment:
This format will crash with the new way to forward the template to the frontend. You need to have the template in a list of objects. You can compare to the other templates once you rebase.